zerver/tests/test_subs.py

```diff
@@ -2128,15 +2132,15 @@ def test_subscriptions_add_notification_default_true(self):
         user_profile.enable_stream_desktop_notifications = True
         user_profile.enable_stream_sounds = True
         user_profile.save()
-        current_stream = self.get_streams(invitee_email)[0]
+        current_stream = self.get_streams(invitee_email, invitee_realm)[0]
         invite_streams = self.make_random_stream_names([current_stream])
         self.assert_adding_subscriptions_for_principal(invitee_email, invitee_realm, invite_streams)
         subscription = self.get_subscription(user_profile, invite_streams[0])
 
         with mock.patch('zerver.models.Recipient.__unicode__', return_value='recip'):
             self.assertEqual(str(subscription),
                              u'<Subscription: '
-                             '<UserProfile: iago@zulip.com <Realm: zulip 1>> -> recip>')
+                             '<UserProfile: {0} <Realm: zulip 1>> -> recip>'.format(self.example_email('iago')))
```

Comment:
Let's use the `%` syntax or `{email}` (aka named format arguments) for this; I think it's a lot more readable.